sdks/builds/wasm.mk

```diff
@@ -0,0 +1,53 @@
+
+WASM_INTERP_CONFIGURE_FLAGS = \
+	--cache-file=$(TOP)/sdks/builds/wasm-interp.config.cache \
+	--prefix=$(TOP)/sdks/out/wasm-interp \
+	--enable-wasm \
+	--enable-interpreter \
+	--disable-mcs-build \
+	--disable-nls \
+	--disable-boehm \
+	--disable-btls \
+	--with-lazy-gc-thread-creation=yes \
+	--with-libgc=none \
+	--disable-executables \
+	--disable-support-build \
+	--disable-visibility-hidden \
+	--enable-minimal=ssa,com,jit,reflection_emit_save,reflection_emit,portability,assembly_remapping,attach,verifier,full_messages,appdomains,security,sgen_remset,sgen_marksweep_par,sgen_marksweep_fixed,sgen_marksweep_fixed_par,sgen_copying,logging,remoting,shared_perfcounters \
+	--host=i386-apple-darwin10
+
+#toolchain code
+.stamp-wasm-toolchain:
+	@./setup-emsdk.sh $(TOP)/sdks/builds/toolchains/emsdk
+	@touch $@
+
+.stamp-wasm-interp-toolchain: .stamp-wasm-toolchain
+	@touch $@
+
+#configure step
+.stamp-wasm-interp-configure: $(TOP)/configure .stamp-wasm-interp-toolchain
+	mkdir -p $(TOP)/sdks/builds/wasm-interp
+	cd $(TOP)/sdks/builds/wasm-interp && source $(TOP)/sdks/builds/toolchains/emsdk/emsdk_set_env.sh && CFLAGS="-Os -g" emconfigure $(TOP)/configure $(WASM_INTERP_CONFIGURE_FLAGS)
+	@touch $@
+
+package-wasm-interp:
+	$(MAKE) -C $(TOP)/sdks/builds/wasm-interp/mono install
+
+
+#custom build rule
+CUSTOM_BUILD_TARGETS += build-wasm-interp
+build-wasm-interp: .stamp-wasm-interp-configure
+	source $(TOP)/sdks/builds/toolchains/emsdk/emsdk_env.sh && make -C wasm-interp
+
+
+
+.PHONY: clean-wasm-interp clean-wasm
+clean-wasm::
+	rm -rf .stamp-wasm-toolchain
+clean-wasm-interp:: clean-wasm
```

Comment:
please unify both `wasm-interp` and `wasm` targets as there is no apparent reason to have `wasm`

Reply:
There's a reason for this.
There are 4 more runtime configs coming down the line: mini aot/runtime, llvm aot/runtime.

This is the only one ready for merging and I feel like keeping the bits for the others made sense.